Flugtag für behinderte ist zu news hunzugefügt

SergejPilot opened the pull request.

SergejPilot committed:
Flugtag für behinderte ist zu news hunzugefügt

jurkan commented on content/news/2016/flugtag-for-invalids.md:
```diff
@@ -0,0 +1,17 @@
+---
+
+Flugtag von Aeroclub Baden-Baden und Fliegergruppe Gaggenau 
+für behinderte und chronisch kranke Kinder und deren Angehörige 
```
Fehlt ein Punkt am Ende

SergejPilot committed:
Punkt hinzugefügt

SergejPilot committed:
Pressreports is added to menu and news are modified

Abschnitt "Presseberichte" ist in config.yaml und ins content
hinzugefügt.
News "flugtag..." und "Kinderspaßfliegen" sind überarbeitet
